Taggers/interface/CollectionDumper.h

```diff
@@ -182,12 +182,37 @@ namespace flashgg {
         auto categories = cfg.getParameter<std::vector<edm::ParameterSet> >( "categories" );
         for( auto &cat : categories ) {
             auto label   = cat.getParameter<std::string>( "label" );
+            //std::cout << "DEBUG CategoryDumper label " << label << std::endl;
             auto systLabel = cat.getParameter<std::string>( "systLabel" );
+            //std::cout << "DEBUG CategoryDumper systlabel " << systLabel << std::endl;
+            auto classname = cat.getParameter<std::string>( "classname" );
```

Comment:
I would actually make this optional.
std::string classname = "";
if( cat.exists("classname") ) { classname =  cat.getParameter<std::string>( "classname" ); }